shims/spark301db/src/main/scala/org/apache/spark/rapids/shims/spark301db/TaskContextTrampoline.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.rapids.shims.spark301db
+
+import org.apache.spark.TaskContext
+import org.apache.spark.memory.TaskMemoryManager
+
+object TaskContextTrampoline {
+
+  def getTaskMemoryManager(): TaskMemoryManager = {
```

Comment:
Why isn't this in TrampolineUtil?

Reply:
Since it was just needed by this one shim, I put it in here, I can move it to common if we prefer?

Reply:
Yes, there may be other cases in the future where we need to get the TaskMemoryManager